controller/app.go

```diff
@@ -171,11 +176,17 @@ func (r *AppRepo) Update(id string, data map[string]interface{}) (interface{}, e
 		}
 	}
 
-	return app, tx.Commit()
+	if err := tx.Commit(); err != nil {
+		return nil, err
+	}
+
+	r.createAppEvent(app.ID)
```

Comment:
Should we not be creating the app events inside the transaction, and throwing an error if it fails?